tokens/src/tests_fungibles.rs

```diff
@@ -287,6 +287,96 @@ fn fungibles_unbalanced_trait_should_work() {
 		});
 }
 
+#[test]
+fn fungibles_balanced_deposit_works() {
+	ExtBuilder::default()
+		.balances(vec![(ALICE, DOT, 100)])
+		.build()
+		.execute_with(|| {
+			let amount = 42;
+			let alice_old_balance = <Tokens as fungibles::Inspect<_>>::balance(DOT, &ALICE);
+			let debt = <Tokens as fungibles::Balanced<_>>::deposit(DOT, &ALICE, amount, Precision::Exact).unwrap();
```

Comment:
I am little bit surprised that this doesn't increase the total issuance of the token. It just sets the balance storage. I thought a deposit out of thin air should always increase the total issuance?

Reply:
this is expected. the total issuance only changed when the `debt` is dropped.